api/lib/infrastructure/http/http-agent.js

```diff
@@ -13,11 +13,13 @@ class HttpResponse {
 
 module.exports = {
   async post({ url, payload, headers }) {
+    const TIMEOUT_MILLISECONDS = 5;
     const startTime = performance.now();
     let responseTime = null;
     try {
       const httpResponse = await axios.post(url, payload, {
         headers,
+        timeout: TIMEOUT_MILLISECONDS,
```

Comment:
Cette suggestion porte plusieurs choix :

* Utiliser une variable d'environnement (comme déjà suggéré par @annemarie35 
* Définir pour cette variable d'environnement le nom `FETCH_TIMEOUT` car pour l'instant on utilise _axios_, mais le présent du web et le futur proche de Node.js n'est pas _axios_ mais la [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) cf. https://blog.logrocket.com/fetch-api-node-js/
* Définir cette variable au début du module, plutôt que dans la fonction, de manière à ce qu'elle ne soit définie et évaluée qu'une seule fois.

Autre remarque à faire en plus de cette suggestion : utiliser ce paramètre `timeout` aussi pour la fonction `get`.

```suggestion
const ms = require('ms');

const TIMEOUT_MILLISECONDS = ms(process.env.FETCH_TIMEOUT || '20s')

module.exports = {
  async post({ url, payload, headers }) {
    const startTime = performance.now();
    let responseTime = null;
    try {
      const httpResponse = await axios.post(url, payload, {
        headers,
        timeout: TIMEOUT_MILLISECONDS,
```